nav2_costmap_2d/plugins/obstacle_layer.cpp

```diff
@@ -658,4 +653,22 @@ ObstacleLayer::reset()
   undeclareAllParameters();
 }
 
+void
+ObstacleLayer::triggerObservationsBufferUpdate()
```

Comment:
I’m not sure this method name describes what’s happening here. Given its doing a single operation over all streams, how about `resetBuffersLastUpdated`?

Reply:
(its also just iterating through a vector, I dont think we need to over complicate it, it could also just be copy pasted twice and I wouldnt argue)